### PR TITLE
deps: bump codespell

### PR DESCRIPTION
**Pull request by pared**
updates:
- [github.com/codespell-project/codespell: v2.1.0 → v2.2.1](https://github.com/codespell-project/codespell/compare/v2.1.0...v2.2.1)

* [ ] ❗ I have followed the [Contributing to DVC](https://dvc.org/doc/user-guide/contributing/core) checklist.

* [ ] 📖 If this PR requires [documentation](https://dvc.org/doc) updates, I have created a separate PR (or issue, at least) in [dvc.org](https://github.com/iterative/dvc.org) and linked it here.

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

### CODE REVIEW

File: tests/unit/dependency/test_params.py [hunk 1]
```diff
@@ -222,7 +222,9 @@ def test_get_hash_missing_param(tmp_dir, dvc):
 
 @pytest.mark.parametrize("param_value", ["", "false", "[]", "{}", "null"])
 def test_params_with_false_values(tmp_dir, dvc, param_value):
-    """These falsy params values should not ignored by `status` on loading."""
+    """
+    These false params values should not be ignored by `status` on loading.
```
Comment (skshetry) [inline tests/unit/dependency/test_params.py:226]: This is correct though, they are a `falsy` values.

Comment (pared) [reply tests/unit/dependency/test_params.py:226]: I guess codespell dictionary is wrong. We can skip this update and report this issue.

Comment (skshetry) [reply tests/unit/dependency/test_params.py:226]: Or, you can just add to the ignore-words-list:
https://github.com/iterative/dvc/blob/122a68398e17f21da1938129aa8cdb21123cb794/.pre-commit-config.yaml#L18-L19
.

Comment (skshetry) [reply tests/unit/dependency/test_params.py:226]: Related: https://github.com/codespell-project/codespell/issues/2444.

**Review by skshetry — APPROVED** [review]

**Pull request merged**